tests/test_warnings.py

```diff
@@ -79,3 +83,98 @@ def __init__(
 
 audobject.config.SIGNATURE_MISMATCH_WARN_LEVEL = \
     audobject.define.SignatureMismatchWarnLevel.STANDARD
+
+
+# Package mismatch
+
+warnings.simplefilter('error')
+
+smaller_version = '0.0.0'
+installed_version = audobject.__version__
+greater_version = '999.9.9'
+
+for level in [
+    audobject.define.PackageMismatchWarnLevel.SILENT,
+    audobject.define.PackageMismatchWarnLevel.VERBOSE,
+    audobject.define.PackageMismatchWarnLevel.STANDARD,
+]:
+    # no version given -> never warn
+    yaml_s = '''
+    $audobject.core.testing.TestObject:
+      name: test
+    '''
+
+    audobject.config.PACKAGE_MISMATCH_WARN_LEVEL = level
+
+    with warnings.catch_warnings():
+        audobject.from_yaml_s(yaml_s)
+
+for level, version, expected in [
```

Comment:
Is it easier this way then using `@pytest.mark.parametrize`?

Reply:
No of course not, changed it.

Reply:
:)